client/state/selectors/is-connected-secondary-network-site.js

```diff
@@ -0,0 +1,43 @@
+/**
+ * External dependencies
+ */
+import { get } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import createSelector from 'lib/create-selector';
+import { isMainSiteOf } from 'state/selectors';
+import { isJetpackSiteSecondaryNetworkSite } from 'state/sites/selectors';
+
+/**
+ * Returns true if site with id equal to siteId is a connected secondary network site and false otherwise
+ * Returns null if with the information available it's not possible to determine if site is a connected secondary site or not
+ *
+ * @param  {Object}    state     Global state tree
+ * @param  {Number}    siteId    The ID of the site we're querying
+ * @return {?Boolean}            Whether site with id equal to siteId is a connected secondary network site
+ */
+export default createSelector(
+	( state, siteId ) => {
+		const isSecondaryNetworkSite = isJetpackSiteSecondaryNetworkSite( state, siteId );
+		if ( isSecondaryNetworkSite == null ) {
```

Comment:
It's always recommended to use the strict comparison operator `===` instead of the equality one `==`.